examples/segmentation_example.ipynb

```diff
@@ -0,0 +1,306 @@
+    "            \"is_particle\": true,\n",
+    "            \"pdb_id\": \"1FA2\",\n",
+    "            \"label\": 2,\n",
+    "            \"color\": [153,  63,   0, 128],\n",
+    "            \"radius\": 65,\n",
+    "            \"map_threshold\": 0.035\n",
+    "        },\n",
+    "        {\n",
+    "            \"name\": \"beta-galactoside\",\n",
+    "            \"go_id\": \"UniProtKB:P00722\",\n",
+    "            \"is_particle\": true,\n",
+    "            \"pdb_id\": \"6X1Q\",\n",
+    "            \"label\": 3,\n",
+    "            \"color\": [ 76,   0,  92, 128],\n",
+    "            \"radius\": 90,\n",
+    "            \"map_threshold\": 0.0578\n",
+    "        },\n",
+    "        {\n",
+    "            \"name\": \"ribosome\",\n",
+    "            \"go_id\": \"GO:0022626\",\n",
+    "            \"is_particle\": true,\n",
+    "            \"pdb_id\": \"6EK0\",\n",
+    "            \"label\": 4,\n",
+    "            \"color\": [  0,  92,  49, 128],\n",
+    "            \"radius\": 150,\n",
+    "            \"map_threshold\": 0.0374\n",
+    "        },\n",
+    "        {\n",
+    "            \"name\": \"thyroglobulin\",\n",
+    "            \"go_id\": \"UniProtKB:P01267\",\n",
+    "            \"is_particle\": true,\n",
+    "            \"pdb_id\": \"6SCJ\",\n",
+    "            \"label\": 5,\n",
+    "            \"color\": [ 43, 206,  72, 128],\n",
+    "            \"radius\": 130,\n",
+    "            \"map_threshold\": 0.0278\n",
+    "        },\n",
+    "        {\n",
+    "            \"name\": \"virus-like particle\",\n",
+    "            \"go_id\": \"GO:0170047\",\n",
+    "            \"is_particle\": true,\n",
+    "            \"pdb_id\": \"6N4V\",            \n",
+    "            \"label\": 6,\n",
+    "            \"color\": [255, 204, 153, 128],\n",
+    "            \"radius\": 135,\n",
+    "            \"map_threshold\": 0.201\n",
+    "        }\n",
+    "    ],\n",
+    "\n",
+    "    \"overlay_root\": \"/tmp/overlay\",\n",
+    "\n",
+    "    \"overlay_fs_args\": {\n",
+    "        \"auto_mkdir\": true\n",
+    "    },\n",
+    "\n",
+    "    \"dataset_ids\": [10441]\n",
+    "}\n",
+    "\"\"\")\n",
+    "\n",
+    "print(proj.pickable_objects)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "CopickPicks(pickable_object_name=ribosome, user_id=data-portal, session_id=74238, len(points)=148) at 0x144474fa0"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pickable_object = proj.pickable_objects[0]\n",
+    "radius = pickable_object.radius\n",
+    "painting_segmentation_name = \"ribosomeTest\"\n",
+    "run = proj.runs[0]\n",
+    "voxel_spacing = 10\n",
+    "tomo_type = \"wbp\"\n",
+    "pick_set = run.get_picks(object_name=\"ribosome\")[0]\n",
```

Comment:
Need to use `cytosolic-ribosome` here if you want to create the project with `copick. from_czcdp_datasets`